### PR TITLE
Better format of installment minimum amount message (Merge after #196)

### PR DESCRIPTION
**Pull request by jacstn**
#### 1. Objective
This PR adds formatting of currency displayed on the minimum installment amount requirement message.

image

Formatting is done according to the merchant's Magento store setup.

This PR also change condition when this message is displayed.
It also adds a condition to this message also when installment payment is active at all.

#### 2. Description of change

Added simple message formatting function.
Added extra condition when the installment warning message is displayed.

#### 3. Quality assurance

**🔧 Environments:**

- **Platform version**: Magento CE 2.1.5.
- **Omise plugin version**: Omise-Magento 2.1.
- **PHP version**: 7.0.16.

**✏️ Details:**

Please do the following tests:
1. check if a properly formatted message is displayed when you have set up the store's currency as THB and order value is below 3000 THB.
2. check if the message is not displayed when 3000 THB is not met, but store's currency is set as non-THB

#### 4. Impact of the change

N/A

#### 5. Priority of change

Normal

#### 6. Additional Notes

N/A

### CODE REVIEW

File: view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js [hunk 1]
```diff
@@ -68,6 +68,17 @@ define(
...
 
+            /**
+             * Get formatted message about installment value limitation
+             *
+             * NOTE: this value should be taken directly from capability object when it is fully implemented.
+             *
+             * @return {string}
+             */
+            getMinimumOrderText: function () {
+                return $.mage.__('Minimum order value is %amount').replace('%amount', this.getFormattedAmount(3000));
```
Comment (jonrandy) [inline view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js]: This minimum amount of 3000 is now in more than one place in this JS file. Should be a constant

Comment (guzzilar) [reply view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js]: I second this.

File: view/frontend/web/template/payment/offsite-installment-form.html [hunk 2]
```diff
@@ -18,10 +18,10 @@
                 </div>
             </div>
         </div>
-        <div data-bind="visible: orderValueTooLow()" class="page messages">
+        <div data-bind="visible: orderValueTooLow() && isActive()" class="page messages">
```
Comment (guzzilar) [inline view/frontend/web/template/payment/offsite-installment-form.html]: Just a minor thing, maybe `isActive() && orderValueTooLow()` would be more familiar to our eyes.
_just personal opinion, feel free to ignore if you don't want to change_

Comment (jacstn) [reply view/frontend/web/template/payment/offsite-installment-form.html]: ok.. it is more logical to check first if it is active at all.

File: view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js (hunk 1 above)
Comment (jacstn) [reply view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js]: ok.. done

File: view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js [hunk 3]
```diff
@@ -22,6 +22,7 @@ define(
         priceUtils,
     ) {
         'use strict';
+        const minimumInstallmentValue = 3000;
```
Comment (jonrandy) [inline view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js]: Not a good name really. It is the minimum value of the purchase for instalment payments to be available, not the minimum instalment value -  an instalment is the monthly payment

Comment (jonrandy) [reply view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js]: Maybe `installmentMinimumPurchaseAmount` or something (although that is a bit long)

Comment (jacstn) [reply view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js]: ok.. changed.. it is bit long.. but temporary.. so can be

**Pull request merged**